R/taxonomy.R

```diff
@@ -69,6 +69,10 @@
 #' @param use_grepl \code{TRUE} or \code{FALSE}: should pattern matching via
 #'   \code{grepl} be used? Otherwise literal matching is used.
 #'   (default: \code{FALSE})
+#'   
+#'@param MARGIN A character value for selecting if data is merged
+#'   row-wise / for features ('rows') or column-wise / for samples ('cols').
+#'   Must be \code{'rows'} or \code{'cols'}.
```

Comment:
This could be hidden. There is not many use-case currently for MARGIN = 2, but good to have support